src/views/Home/index.tsx

```diff
@@ -14,7 +13,6 @@ import Footer from './components/Footer'
 import CakeDataRow from './components/CakeDataRow'
 import { WedgeTopLeft, InnerWedgeWrapper, OuterWedgeWrapper, WedgeTopRight } from './components/WedgeSvgs'
 import UserBanner from './components/UserBanner'
-import FarmAuctionsBanner from './components/FarmAuctionsBanner'
```

Comment:
remove the component if it's not used anymore

Reply:
Going to be used again in a week 👍

Reply:
I've deleted and then re-added banners so many times. Plz spare this lil guy